chain/actions/src/lib.rs

```diff
@@ -110,19 +111,19 @@ pub mod redeem;
 /// Contains all actions that a node can execute on-chain.
 #[derive(Debug, Clone)]
 pub struct ChainActions<Db: HoprCoreEthereumDbActions + Clone> {
-    me: Address,
+    me: ChainKeypair,
     db: Arc<RwLock<Db>>,
     tx_sender: ActionSender,
 }
 
 impl<Db: HoprCoreEthereumDbActions + Clone> ChainActions<Db> {
     ///! Creates new instance.
-    pub fn new(me: Address, db: Arc<RwLock<Db>>, tx_sender: ActionSender) -> Self {
+    pub fn new(me: ChainKeypair, db: Arc<RwLock<Db>>, tx_sender: ActionSender) -> Self {
```

Comment:
Should be `me: &ChainKeypair`

Reply:
postponed for https://github.com/hoprnet/hoprnet/issues/6049